dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Register/IttProvider.cshtml

```diff
@@ -47,14 +48,9 @@
 
                     <govuk-radios-item value="True">Yes
                         <govuk-radios-item-conditional>
-                            <govuk-select asp-for="IttProviderName">
-                                <govuk-select-label class="govuk-label--s"/>
-                                <govuk-select-item value=""></govuk-select-item>
-                                @foreach (var ittProviderName in Model.IttProviderNames ?? Enumerable.Empty<string>())
-                                {
-                                    <govuk-select-item value="@ittProviderName">@ittProviderName</govuk-select-item>
-                                }
-                            </govuk-select>
+                            <govuk-input asp-for="IttProviderName">
+                                <govuk-input-label class="govuk-label--s"/>
+                            </govuk-input>
```

Comment:
Some strange behaviour I wasn't able to fix: when returning to this page from check-answers with a itt-provider that is not in the dropdown list, the dropdown is initially open showing that custom entry as a result. (Will attach image)

Reply:
@wilpow-github I've come across this before. I'll see if I can remember if there was a fix, and if so, what it is.